src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -142,19 +153,17 @@ export class PaymentPageView extends View<CartDataType> {
 
             numberInput.value = numberInput.value.replace(/[^+\d]/g, '').substring(0, 16);
             if (numberInput.value.match(/^\+\d{9}/g)) {
-                numberInput.classList.add('valid');
-                numberInput.classList.remove('invalid');
-                validTell = true;
+                valid.validTell = true;
+                setInputValidity(numberInput, valid.validTell);
             } else {
-                numberInput.classList.remove('valid');
-                numberInput.classList.add('invalid');
-                validTell = false;
+                valid.validTell = false;
+                setInputValidity(numberInput, valid.validTell);
             }
         })
 
         // валидация card number
         const cardNumberInput = document.querySelector<HTMLInputElement>(".card-details__card-number");
-        const cardsImg = document.querySelector<HTMLElement>(".cards__img");
+        const cardsImg = document.querySelector(".cards__img") as HTMLElement | null;
 
         cardNumberInput?.addEventListener('input', (event: Event) => {
             let arrCardNumber = Array.from(cardNumberInput.value)
```

Comment:
If you need only first `el` from string could you try to find another solution to get it?